app/views/layouts/application.html.erb

```diff
@@ -9,18 +30,107 @@
+                            </li>
+                            <li>
+                                <%= link_to "Lectures", :controller => :lectures, :action => :index %>
+                            </li>
+                            <li>
+                                <%= link_to "Responses", :controller => :responses, :action => :index %>
+                            </li>
+                            <li>
+                                <%= link_to "Course to lecture junction", :controller => :course_to_lecture_junctions, :action => :index %>
+                            </li>
+                            <li>
+                                <%= link_to "Question to tag junction", :controller => :question_to_tag_junctions, :action => :index %>
+                            </li>
+                            <li>
+                                <%= link_to "Question to set junction", :controller => :question_set_junctions, :action => :index %>
+                            </li> -->
+                        <% else %>
+                            <li>
+                                <a href="#"></i>Courses<span class="fa arrow"></span></a>
+                                <ul class="nav nav-second-level">
+                                    <% current_user.courses.each do |course| %>
+                                        <li>
+                                            <%= link_to course.name, course, { method: :get, controller: :course} %>
+                                        </li>
+                                    <% end %>
+                                    <% if current_user.is_professor %>
+                                        <li>
+                                            <!--<i class="fa fa-edit fa-fw"></i>-->
+                                            <%= link_to "Create New Course", new_course_path, {method: :get, controller: :course} %>
+                                        </li>
+                                    <% end %>
+                                </ul>
+                            <!-- /.nav-second-level -->
+                            </li>
+                            <li>
+                                <%= link_to "Questions", :controller => :questions, :action => :index %>
+                            </li>
+                        <% end %>
+                    </ul>
+                </div>
+                <!-- /.sidebar-collapse -->
+            </div>
+            <!-- /.navbar-static-side -->
+        </nav>
     <div id="main-container" class="container">
-      <div class="row col-xs-12 well text-center">
-        <%= render 'shared/header' %>
-      </div>
-      <div class="row">
-        <%= render 'shared/sidebar' %>
-        <div class="col-xs-9">
-          <%= yield %>
-        </div>
-      </div>
+        <%= yield %>
     </div>
+    <!-- jQuery -->
+    <script src="/vendor/jquery/jquery.min.js"></script>
```

Comment:
We should put each of these files in the assets folder and let them be automatically be put in by rails. Also we may not need jquery and bootstrap.min.js, I think rails is already putting them in here